refs/etsi.json

```
@@ -60626,11 +60685,11 @@
         "source": "https://dashifspecref.blob.core.windows.net/etsi/etsi.json?st=2019-04-01T00%3A00%3A00Z&se=2039-04-01T00%3A00%3A00Z&sp=rl&sv=2018-03-28&sr=b&sig=J%2Fe40FI%2FkhFWRZMUEzmbr6pkNwukw6ChdjsifRaQ%2F%2Bg%3D"
     },
     "etsi-ts-103-548": {
-        "href": "https://www.etsi.org/deliver/etsi_ts/103500_103599/103548/01.02.01_60/ts_103548v010201p.pdf",
+        "href": "http://www.etsi.org/deliver/etsi_ts/103500_103599/103548/01.02.01_60/ts_103548v010201p.pdf",
         "title": "SmartM2M; SAREF reference ontology patterns",
         "status": "Published",
         "publisher": "ETSI",
-        "date": "2024"
+        "date": "2024-01"
```

A mere `2024` works for a human readable `date`, but `2024-01` does not. `rawDate` should be used instead:

```suggestion
        "rawDate": "2024-01"
```